### PR TITLE
Average Annual Equivalent Terms 

### PR DESCRIPTION
**Pull request by rnugent3**
I wrote out the math. I don't know how to interact with IContainResults. I'll write tests after I figure that out.

### CODE REVIEW

File: fda-model/alternatives/Alternative.cs [hunk 1]
```diff
@@ -7,23 +7,60 @@ public class Alternative
...
         }
         public IList<metrics.IContainResults> ComputeEEAD(interfaces.IProvideRandomNumbers rp, Int64 iterations, double discountRate){
             //probably instantiate a rng to seed each impact area differently
-            IList<metrics.IContainResults> currentEAD = _currentYear.Compute(rp,iterations);
-            IList<metrics.IContainResults> futureEAD = _futureYear.Compute(rp, iterations);
-            //discoiunt future ead
-            IList<metrics.IContainResults> discountedEAD = discount(futureEAD,discountRate, _futureYear.Year - _currentYear.Year);
-            //combine the future and current ead
-            return currentEAD;//not right, fix this.
+            IList<metrics.IContainResults> baseEAD = _currentYear.Compute(rp,iterations);
```
Comment (rnugent3) [inline fda-model/alternatives/Alternative.cs:20]: @HenryGeorgist could you please point out to me how I can access baseEAD and mlfEAD? I do not see any available properties or methods that can get a double from the IList<metrics.IContainResults> object.

Comment (rnugent3) [reply fda-model/alternatives/Alternative.cs:20]: I'd like to get some unit testing running on this code asap.

Comment (HenryGeorgist) [reply fda-model/alternatives/Alternative.cs:20]: sure for reference the test here shows it in the other simulation tests:
https://github.com/HydrologicEngineeringCenter/fda-model/blob/b968c90ccff9c71400758caceb74a03c7f3054e4/fda-model-test/simulationShould.cs#L42

it should be something like baseEAD.MeanEAD("Total");

**Review by HenryGeorgist — APPROVED** [review]
looks good - i think this is ready to merge?

**Comment by rnugent3** [comment]
As soon as I finish this test. If all goes well, I’ll push in 20min.

**Pull request merged**